app/assets/javascripts/global-bar-init.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+window.GOVUK = window.GOVUK || {}
+
+// Bump this if you are releasing a major change to the banner
+// This will reset the view count so all users will see the banner, even if previously seen
+var BANNER_VERSION = 3;
+
+var globalBarInit = {
+  getBannerVersion: function() {
+    return BANNER_VERSION
+  },
+
+  blacklistedUrl: function() {
+    return /^\/register-to-vote|^\/done|^\/brexit|^\/get-ready-brexit-check/.test(window.location.pathname)
+  },
+
+  setBannerCookie: function() {
+    var expiryDate = new Date(Date.now()+72576e5).toUTCString()
+    var value = JSON.stringify({count: 0, version: globalBarInit.getBannerVersion()})
+    var cookieString = "global_bar_seen=" + value + "; expires=" + expiryDate + ";"
+
+    document.cookie=cookieString
```

Comment:
Do you think it's worth using the `window.GOVUK.setCookie` helper to set the cookies here?

Reply:
I did this to start with but was getting errors about `GOVUK.getCookie`/`GOVUK.setCookie` not being available. I think because this JS is specifically being pulled in earlier than other JS files, we perhaps can't rely on the cookie helper functions being available at that time?

Reply:
Argh, that's frustrating!

Reply:
@injms Having had a quick look, I don't think it actually needs to be pulled in earlier, so it's possible we could move it into `global-bar.js` and use the helper functions. 

I had a lot of trouble with tests when I did that though, so I'm thinking we can keep this as a first pass and iterate in future. It would be good to get this code merged first, so any urgent changes needed post-election can be done with more confidence.

Reply:
Yep, makes complete sense - thanks for explaining 👍